mempool/src/lib.rs

```diff
@@ -2,7 +2,6 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #![forbid(unsafe_code)]
-#![deny(missing_docs)]
```

Comment:
why we had to remove this?

Reply:
Just add in documentation/comments and you won't get clippy errors